frontend/app/src/helpers.tsx

```diff
@@ -17,6 +17,11 @@ export function satToUsd(amount) {
   return returnValue;
```

Comment:
Just a heads up for this file in general I think we can change the extension to just `ts` since we don't return any `tsx` objects in here